cloudferrylib/os/image/glance_image.py

```diff
@@ -293,6 +296,10 @@ def convert(self, glance_image, cloud):
         gl_image = {
             k: w for k, w in glance_image.to_dict().items(
             ) if k in CREATE_PARAMS}
+        gl_image_default_value = {
+            k: None for k in gl_image.keys()
+            if k not in CREATE_PARAMS}
+        gl_image.update(gl_image_default_value)
```

Comment:
I think I understand what you wanted to do here, but you have mistake here: Variable `gl_image_default_value` will always be assigned to empty dictionary.  You could do the same (lines 296-302) with more compact and easy to understand code:

    image_dict = glance_image.to_dict()
    gl_image = {k: image_dict.get(k) for k in CREATE_PARAMS}